front/lib/resources/content_fragment_resource.ts

```diff
@@ -365,13 +367,68 @@ async function renderFromFileId(
   } else {
     const content = await getProcessedFileContent(workspace, fileId);
 
+    const messageWithFullContent = {
+      role: "content_fragment" as const,
+      name: `inject_${contentType}`,
+      content: [
+        {
+          type: "text",
+          text: `<attachment type="${contentType}" title="${title}">\n${content}\n</attachment>`,
+        } as Content,
+      ],
+    };
+
+    if (contentType !== "text/csv") {
+      return new Ok(messageWithFullContent);
+    }
+
+    // For CSV files we have a custom rule:
+    // We render completely the content if the csv is small or a snippet if the csv is big.
```

Comment:
When you say "completely", it means the same as what we do for any other file right "completely" if it fits ?

Reply:
Yes it means the full content, I edited the comment to clarify

Reply:
I would go for 0% for now as it's easier to explain to users. CSV are just not visible by models?

Reply:
For now we haven't solved @lasryaric's "bosser sur des CSVs" use case, so that would be a pretty big regression for some users no ?

Reply:
The solution to that is to table query on them but that's a larger project?

Reply:
I could be convinced if the CSV is really small since the model will have to echo it entirely but then users won't understand when it works or not.

We can prompt the model to say that it can do only one thing on CSV in the interim?

Reply:
For sure a larger project, but probably not insanely large either